includes/payment-methods/class-wc-stripe-ocs-payment-gateway.php

```
@@ -125,6 +132,29 @@ public function javascript_params(): array {
+	 */
+	protected function get_adaptive_pricing_saved_token_payment_methods(): array {
+		$map = [];
+
+		foreach ( $this->get_tokens() as $token ) {
+			if ( $token instanceof WC_Payment_Token_CC ) {
+				$map[ $token->get_id() ] = $token->get_token();
+			}
+		}
+
+		return $map;
+	}
```

PHPMD (2.15.0)

[error] The method get_adaptive_pricing_saved_token_payment_methods is not named in camelCase. (undefined)

(CamelCaseMethodName)